nano/node/active_transactions.cpp

```diff
@@ -932,4 +917,35 @@ nano::recently_confirmed_cache::entry_t nano::recently_confirmed_cache::back ()
 {
 	nano::lock_guard<nano::mutex> guard{ mutex };
 	return confirmed.back ();
+}
+
+/*
+ * class recently_cemented
+ */
+
+nano::recently_cemented_cache::recently_cemented_cache (std::size_t max_size_a) :
```

Comment:
Why not put the class recently_cemented_cache in its own file?
It is reusable code that could, in theory, be used by other classes too.